tests/test_fields.py

```diff
@@ -39,82 +40,82 @@ def test_can_parse_unquoted_property_names_with_extra_spaces(self):
         assert FieldForTesting(node).name == "Propertyname"
-        assert FieldForTesting(node).date_format == "%d %y %M"
+        assert FieldForTesting(node).date_format == "E yy MM"
 
     def test_can_parse_unquoted_date_format(self):
-        node = ' DOCPROPERTY "Propertyname" \\@ dd-yy-mm \\* MERGEFORMAT '
+        node = ' DOCPROPERTY "Propertyname" \\@ DDD-yy-MM \\* MERGEFORMAT '
         assert FieldForTesting(node).name == "Propertyname"
-        assert FieldForTesting(node).date_format == "%d-%y-%M"
+        assert FieldForTesting(node).date_format == "E-yy-MM"
 
     def test_can_parse_unquoted_date_format_with_spaces(self):
-        node = ' DOCPROPERTY "Propertyname" \\@ dd yy mm \\* MERGEFORMAT '
+        node = ' DOCPROPERTY "Propertyname" \\@ dddd yy MMMM \\* MERGEFORMAT '
         assert FieldForTesting(node).name == "Propertyname"
-        assert FieldForTesting(node).date_format == "%d %y %M"
+        assert FieldForTesting(node).date_format == "EEEE yy MMMM"
 
     def test_can_parse_unquoted_date_format_with_extra_spaces(self):
-        node = ' DOCPROPERTY "Propertyname"   \\@ dd yy mm   \\* MERGEFORMAT'
+        node = ' DOCPROPERTY "Propertyname"   \\@ dd yy MM   \\* MERGEFORMAT'
         assert FieldForTesting(node).name == "Propertyname"
-        assert FieldForTesting(node).date_format == "%d %y %M"
+        assert FieldForTesting(node).date_format == "dd yy MM"
 
 
 class TestFieldDateFormatMapping(object):
 
     def test_correctly_maps_simple_date(self):
         date = datetime(2020, 11, 19)
 
-        assert word_to_python_date_format('yy/MM/dd') == '%y/%m/%d'
-        assert word_to_python_date_format('YY/MM/DD') == '%y/%m/%d'
-        assert date.strftime(
-            word_to_python_date_format('yy/MM/dd')) == '20/11/19'
+        assert word_to_python_date_format('yy/MM/dd') == 'yy/MM/dd'
+        assert word_to_python_date_format('YY/MM/DD') == 'yy/MM/dd'
+        assert format_datetime(
+            date, word_to_python_date_format('yy/MM/dd')) == '20/11/19'
 
-        assert word_to_python_date_format('yyyy/MM/dd') == '%Y/%m/%d'
-        assert word_to_python_date_format('YYYY/MM/DD') == '%Y/%m/%d'
-        assert date.strftime(
-            word_to_python_date_format('YYYY/MM/DD')) == '2020/11/19'
+        assert word_to_python_date_format('yyyy/MM/dd') == 'yyyy/MM/dd'
+        assert word_to_python_date_format('YYYY/MM/DD') == 'yyyy/MM/dd'
+        assert format_datetime(
+            date, word_to_python_date_format('YYYY/MM/DD')) == '2020/11/19'
 
     def test_correctly_maps_date_padding(self):
         date = datetime(2001, 2, 4)
 
-        assert word_to_python_date_format('yy/MM/dd') == '%y/%m/%d'
-        assert date.strftime(
-            word_to_python_date_format('yy/MM/dd')) == '01/02/04'
+        assert word_to_python_date_format('yy/MM/dd') == 'yy/MM/dd'
+        assert format_datetime(
+            date, word_to_python_date_format('yy/MM/dd')) == '01/02/04'
```

Comment:
In these tests you see that the same format strings in word are still supported, lead to different python formatting strings but in the end to the same formatting of the date.